tests/integration/test_github/test_configuration/data/config_templates/config_invalid_configuration.yaml

```diff
@@ -0,0 +1,45 @@
+- sections:
+    - section: github
+      elements:
+        - enabled:
+            value: 'ENABLED'
+        - only_future_events:
+            value: 'ONLY_FUTURE_EVENTS'
+        - interval:
+            value: 'INTERVAL'
+        - curl_max_size:
+            value: 'CURL_MAX_SIZE'
+        - time_delay:
+            value: 'TIME_DELAY'
+        - api_auth:
+            elements:
+              - org_name:
+                  value: 'ORG_NAME'
+              - api_token:
+                  value: 'API_TOKEN'
+        - api_parameters:
+            elements:
+              - event_type:
+                  value: 'EVENT_TYPE'
+
+    - section: sca
+      elements:
+        - enabled:
+            value: 'no'
+
+    - section: rootcheck
+      elements:
+        - disabled:
+            value: 'yes'
+
+    - section: syscheck
+      elements:
+        - disabled:
+            value: 'yes'
+
+    - section: wodle
+      attributes:
+        - name: syscollector
+      elements:
+        - disabled:
+            value: 'yes'
```

Comment:
Add an empty line at the end of the file

Reply:
Change applied with [this commit](https://github.com/wazuh/wazuh/pull/17621/commits/42030e38bc82fa2fb6e842adf8abee6b919cfeef)